tests/acceptance/api_test.go

```diff
@@ -1136,72 +1141,8 @@ func TestApi(t *testing.T) {
 					assert.Equal(t, 1, len(results), "one lease should be returned")
 					assert.Equal(t, "Inactive", results[0]["leaseStatus"])
 
-					t.Run("STEP: Recreate lease against same account", func(t *testing.T) {
```

Comment:
This is now covered in a dedicated test, that attempts to create/destroy/re-create leases several times.